01-start/03-begin.html.md

```diff
@@ -565,7 +560,7 @@ As you now have all the tools and knowledge required to be able to create the bl
 
 Congratulations! You now possess all the foundations required to be able to write amazing and powerful web applications like those already in our [Showcase](/docpad/showcase). To recap, you now know how to:
 
-- write documents in any language, markup, pre-processor, templating engine, whatever you wish, by installing the necessary plugin for it and changing the extensions of the document
+- write `render` files in any language, markup, pre-processor, templating engine, whatever you wish, by installing the necessary plugin for it and changing the extensions of the document
```

Comment:
Considering the mass renames from `documents` to `render files`; perhaps we need to update http://docpad.org/docs/overview to mention which model types are being used for the various directories, as there are two model types inside DocPad: Documents (files that are rendered; render files, layout files), as well as Files (files that are not rendered, such as those in static). The Document class extends the File class.

Such a thing would also be needed, for http://docpad.org/docs/meta-data to make sense given these changes.